dynet/nodes.cc

```diff
@@ -1275,6 +1275,24 @@ void NoBackprop::backward_dev_impl(const MyDevice & dev,
 DYNET_NODE_INST_DEV_IMPL(NoBackprop)
 
 template<class MyDevice>
+void FlipGradient::forward_dev_impl(const MyDevice & dev, const vector<const Tensor*>& xs, Tensor& fx) const {
+  fx.d = xs[0]->d;
```

Comment:
There should be no need for this line, as dim_forward will be sure to set the dimensions of fx to the proper size.